### DISCUSSION TITLE
【问题】不要使用mumu12的vulkan渲染模式，会产生OCR识别错误（刷紧急委托换船会漏装备）、截图超时等问题

### DISCUSSION THREAD

**Discussion by soloopooo** [LmeSzinc/AzurLaneAutoScript#4364] [Show and tell]
我查了以下目前的issue, 部分issue显示用mumu截图超时和其他一些包括OCR识别失败（表现为刷紧急委托换船会漏装备）等的奇奇怪怪的问题，之前我也有过这种问题。
我挂机15小时左右必然会报错截图超时。
后来将mumu模拟器的vulkan渲染模式改成directX就不会再出现什么截图超时或OCR识别失败的问题了。

**Comment by soloopooo** [comment]
另外vulkan模式还有可能导致内存泄漏，我给mumu反馈过bug至今没回复我

**Reply by ye0577** [reply]
老哥 我切换了dx 还是一样会漏装备 有解决办法吗
